src/services/proposals-votes.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Inject, Injectable } from '@nestjs/common';
+
+import { Repository } from 'typeorm';
+
+import { ProposalsVotesEntity } from '@app/database';
+import { LumConstants, LumUtils } from '@lum-network/sdk-javascript';
+import { LumBech32PrefixValAddr } from '@lum-network/sdk-javascript/build/constants';
+
+@Injectable()
+export class ProposalsVotesService {
+    constructor(@Inject('PROPOSALS_VOTES_REPOSITORY') private readonly _repository: Repository<ProposalsVotesEntity>) {}
+
+    getById = async (proposalId: number): Promise<ProposalsVotesEntity> => {
```

Comment:
This should be named `getByProposalID` and we should have a `getById` method that takes a string (composite key)